### PR TITLE
[pro#413] expand new signup welcome message

### PR DESCRIPTION
**Pull request by lizconlan**
screen shot 2017-11-21 at 16 06 36

Closes mysociety/alaveteli-professional#413

### CODE REVIEW

**Comment by lizconlan** [comment]
ToDo - break the message into 2 strings so as not to have HTML in translation strings.

**Comment by garethrees** [comment]
Oops hang on – just tried with WDTK theme and got:

screen shot 2017-11-22 at 17 08 37

File: app/views/alaveteli_pro/subscriptions/_signup_message.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p><%= _('Welcome to {{pro_site_name}}!',
+         pro_site_name: AlaveteliConfiguration.pro_site_name) %>
+%></p>
```
Comment (garethrees) [inline app/views/alaveteli_pro/subscriptions/_signup_message.html.erb]: Double closing erb tag here

Comment (lizconlan) [reply app/views/alaveteli_pro/subscriptions/_signup_message.html.erb]: ugh, copy and paste fail - thank you!

**Pull request merged**